src/Router/Router.js

```diff
@@ -96,6 +98,10 @@ class Router extends Component {
             <CharProfilePage />
           </Route>
 
+          <Route path="/quiz" exact>
```

Comment:
Was this implemented just for testing, or do you think we will use the actual route from here instead of calling the component from inky?